### ISSUE TITLE
Purchase a hero

### ISSUE THREAD

**Issue by yuulye** [fifa-mobile/discord-bot#19]
@fifa-mobile/nubs what about his option? currently $hero will pick you a random hero

let's say $100 for a pick? hahaha

**Comment by itzsrs** [comment]
😎

**Comment by yuulye** [comment]
@itzsrs so ok? $100 for a hero purchase?

**Comment by Pradeep5986** [comment]
No

**Comment by yuulye** [comment]
@Pradeep5986 yeah, so you can play coins much, although it can be easily killed hahaha

**Comment by Pradeep5986** [comment]
Nice,let's try it in test bot

**Comment by juve938383** [comment]
I think we should focus economy on fm commands and keep other ones for fun.FM commands should be our focus imo and other ones just additions for fun.

**Comment by itzsrs** [comment]
Some of the Ben 10 fighters were awesome. Like Ultimate Humungousaur and Swampfire

**Comment by Pradeep5986** [comment]
Did,they were added
